katas/metal-slug/daniel/Weapon.js

```diff
@@ -0,0 +1,34 @@
+export class Weapon {
+    constructor () {
+        this.name = '';
+        this.attackPower = 0;
+        this.multipleEnemys = false;
+    }
+
+    attack ( enemies ) {
+        if ( Array.isArray(enemies) ){
+            // Several enemies were input, just reduce the pointsOfHealth of the first one
```

Comment:
Ok, not bad. The risk I see with this approach is that you are opening the possibility for `enemies` to be either an array or a non-array object. In such case, I think this could be handled better with some sort of classes hierarchy